lib/dbservice_web/plugs/domain_whitelist_plug.ex

```diff
@@ -25,7 +25,7 @@ defmodule DbserviceWeb.DomainWhitelistPlug do
 
     allowed_domains =
       if is_nil(whitelisted_domains),
-        do: ["localhost"],
+        do: ["localhost", "154b-103-55-63-187.in.ngrok.io"],
```

Comment:
It can be done but there is no default way of doing that. In order to achieve this, we need to add a custom third-party library which may effect the existing case